weekly-updates/project-lead-meetings/10_14_18.md

```diff
@@ -130,10 +130,13 @@ _TO DO (for all of next semester)_:
 
 ## snackru
 
-_UPDATES_:
+_UPDATES_: nothing much, sorry. Architects made a some progress in the weeks leading up to HackRU 
 
-_QUESTIONS_:
+_QUESTIONS_: ??
 
-_TO DO (for all of next semester)_:
+_TO DO (for all of next semester)_: 
+    -make a good frontend
+    -add functionality (end goal is that food table volunteers are self-sufficient & can use the app guide them)
+    - (as a post-mortem comment) I have to take a a more hands-on approach to leading this project or it will never get done.  
```

Comment:
Um... wdym more hands-on?

Do whatever you think is necessary to get it done, I guess. (But, ofc, give architects a chance to learn as much as possible.)